internal/devices/update.go

```diff
@@ -123,12 +129,11 @@ func (c *Client) Update() *cobra.Command {
 	updateDeviceCmd.Flags().StringVarP(&description, "description", "d", "", "Adds or updates the description for the device.")
 	updateDeviceCmd.Flags().StringVarP(&userdata, "userdata", "u", "", "Adds or updates the userdata for the device.")
 	updateDeviceCmd.Flags().StringVarP(&userdataFile, "userdata-file", "", "", "Path to a userdata file for device initialization. Can not be used with --userdata.")
-	updateDeviceCmd.Flags().BoolVarP(&locked, "locked", "l", false, "Locks or unlocks the device for future changes (<true|false>).")
+	updateDeviceCmd.Flags().BoolSliceP("locked", "l", []bool{}, "Locks or unlocks the device for future changes (<true|false>).")
```

Comment:
I initially tried `BoolP`, but even when I sent `--locked false`, `GetBool` would return `true`. I think it was treating `--locked` like an argument that takes no parameters, in which case `false` would have been an `args[]` argument. Sending additional `foo` args on the command line doesn't trigger any errors, which backs this theory.

Reply:
Looks like we would need the following to generate an error in that condition:
```
updateDeviceCmd.Args = cobra.NoArgs
```

Reply:
I'll add that in this PR. 

I can see us wanting to one-shot that change across the repo since we don't accept args for most (if any) commands.

Reply:
It seems cobra requires `=` to set a value for boolean params: https://github.com/spf13/cobra/issues/1606

In that case the syntax would be `--locked=false`; we would still have to update the command logic to correctly handle unlocking (it currently only does something if `locked` is true).  An alternative would be a second `--unlocked` boolean flag that is mutually exclusive with `locked` (my personal preference and seems to be the recommended style for cobra commands).  We've also talked about switching to a string param, although that would require specifying the value `--locked <value>` instead of using the shorthand `--locked`.

Reply:
I could also go that way. The documentation previously said that the field was boolean and that `--locked <true|false>` was the way to toggle it. This PR upholds the documentation.

In practice, `--locked=<true|false>`, `--locked <true|false>`, and `--locked` all had the same affect previously.
I could see taking the alternative approach, introducing a new `--unlocked`.

Reply:
The usage is hard-coded in the `Use` property of the subcommand and is not automatically generated based on the calls to `updateDeviceCmd.Flags()`, so the docs don't necessarily accurately reflect usage.  If you look at the usage for the `--always-pxe` flag, that matches the documented behavior/expectations of cobra boolean flags and establishes the expectation that a boolean flag value must be set with `=`.  Given that there's already a flag in this subcommand that takes `--<flag>=true|false` there's precedent for aligning `--locked` with that rather than introducing a separate flag.